scripts/ci-check-to-ensure-sql-is-in-lowercase.mjs

```diff
@@ -0,0 +1,98 @@
+import fs from "fs";
+import path from "path";
+
+const regex = /```sql\n([\s\S]*?)\n```/g;
```

Comment:
`m` enables "multiline" mode where `^` and `$` match beginning/end of line (rather than of string); we can then check that the line starts/ends with ```` ``` ````:

```suggestion
const regex = /^```sql\n([\s\S]*?)\n```$/gm;
```